sdk/iotcentral/Azure.ResourceManager.IotCentral/src/Generated/Models/AppTemplateLocations.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.IotCentral.Models
+{
+    /// <summary> IoT Central Application Template Locations. </summary>
+    public partial class AppTemplateLocations
+    {
+        /// <summary> Initializes a new instance of AppTemplateLocations. </summary>
+        internal AppTemplateLocations()
+        {
+        }
+
+        /// <summary> Initializes a new instance of AppTemplateLocations. </summary>
+        /// <param name="id"> The ID of the location. </param>
```

Comment:
is this same as `AzureLocation` or not?

Reply:
Hi @chgennar, can you help to confirm if this class `AppTemplateLocations` is same as the class [AzureLocation](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/AzureLocation.cs) that we definied in Azure.Core?

Reply:
@Yao725, I believe they are the same. Is there a common type that can be used in the swagger spec for this?

Reply:
Maybe it is possible to use `AzureLocation` here and to account for the difference in property name through the serializer?

Reply:
My bad, I double checked this class. I think what we want to do here is change the type of the property `id` to `AzureLocation` instead of string if it really represents an `AzureLocation` like `centralus`.

Sorry for the wrong description of the problem before.

Reply:
Yes, it looks weird, so let's keep this property unchanged for now.